src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(function (resolve, reject) {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+const promise2 = new Promise(function (resolve, reject) {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
+});
+
+promise1
+  .then((data) => {
+    const div = document.createElement('div');
```

Comment:
You have almost identical logic for creating success and error messages. Why don't we create a helper function like 'generateMessage'?